app/Http/Controllers/Admin/Nodes/NodeViewController.php

```diff
@@ -118,13 +119,23 @@ public function allocations(Request $request, Node $node)
 
         $this->plainInject(['node' => Collection::wrap($node)->only(['id'])]);
 
-        return $this->view->make('admin.nodes.view.allocation', [
-            'node' => $node,
-            'allocations' => Allocation::query()->where('node_id', $node->id)
-                ->groupBy('ip')
-                ->orderByRaw('INET_ATON(ip) ASC')
-                ->get(['ip']),
-        ]);
+        if (DB::getDriverName() === 'pgsql') {
+            return $this->view->make('admin.nodes.view.allocation', [
+                'node' => $node,
+                'allocations' => Allocation::query()->where('node_id', $node->id)
+                    ->groupBy('ip')
+                    ->orderByRaw('ip ASC')
+                    ->get(['ip']),
```

Comment:
This can be simplified by making a simple ternary: 
```php
->orderByRaw(DB::getDriverName() === 'pgsql' ? 'ip ASC' : 'INET_ATON(ip) ASC')
```
If you do that, you can remove the if statement and you won't have to repeat the same code twice.

Reply:
Thanks, will change to ternary where possible.